### ISSUE TITLE
Attachments get lost when fetching signed mails

### ISSUE THREAD

**Issue by domshifter** [pmengal/MailSystem.NET#32]
When i try to fetch a signed mail via the **Pop3Client** or **Imap4Client**, i won't get any attachments. Getting the attachments from the same mail, without signing works as expected.

Is there a solution for this problem or is this a known issue? Any help would be greatly appreciated!

**Comment by reinaldocoelho** [comment]
Hi @domshifter !
I never heard about this issue.
Can you give to us the steps to reproduce the problem?

**Comment by domshifter** [comment]
Hi @reinaldocoelho ho, thank you for your fast reply!

We used Comodo to generate a certificate which we then used to sign the mails. 

I tested three scenarios:

1. Send a mail from my dev machine, then fetch it with my application which also runs on my machine. This works (maybe because i got the certificate installed on my machine which i used to sign the mails?)

2. Send a mail from my dev machine, then fetch it with my application on another dev machine which is in the same domain. This also works.. (maybe because the two pc's are in the same domain?)

3. Send a mail from outside our domain, then fetch it on my local running application. In this case, i'm not able to get the attachments.

Our code looks something like this:

```
_client = new Imap4Client();
_client.ConnectSsl(settings.ServerName, settings.Port);
```
```
var inbox = _client.SelectMailbox("inbox"); 
var messages = inbox.SearchParse("ALL");

foreach (var message in messages)
{
    foreach (MimePart attachment in message.Attachments)
    {
        //process attachment
    }
}
```

```
_client.Disconnect();
```
